lua/refactoring/config.lua

```diff
@@ -6,6 +6,9 @@ local config = {
         lua = {
             -- cmd = [[ !stylua % ]],
         },
+        go = {
+
```

Comment:
perhaps gofmt could be used here.  I assume its the "defacto" format lib.  You could add a quick check before executing it so it doesn't vomit errors on all golangers.

Reply:
Tried to add gofmt, but ran into similar issue I assumed you ran into with stylua where it's vomiting on the `%` while testing. Tried to run manually but it still wasn't formatting either... Let me know if you want me to open another issue to look into this as well.

Reply:
yeah, it has to do with the fact that you are editing the file, with changes, outside of vim.  Causing two different file trees.